ledger/src/blockstore_processor.rs

```diff
@@ -155,7 +155,7 @@ fn execute_batch(
         vec![]
     };
 
-    let (tx_results, balances) = batch.bank().load_execute_and_commit_transactions(
+    let (tx_results, balances) = bank.load_execute_and_commit_transactions(
```

Comment:
Change to use the `bank` parameter instead of bank reference on the `batch`, done to avoid immutable & mutable references to batch.
See no reason these should be different, but might be missing some additional context.